apps/client/src/common/components/input/auto-text-area/AutoTextArea.tsx

```diff
@@ -1,27 +1,25 @@
-import { useEffect, useRef } from 'react';
+import { RefObject, useEffect } from 'react';
 import { Textarea, TextareaProps } from '@chakra-ui/react';
 // @ts-expect-error no types from library
 import autosize from 'autosize/dist/autosize';
 
-export const AutoTextArea = (props: TextareaProps) => {
-  const { value } = props;
-
-  const ref = useRef<HTMLTextAreaElement>(null);
+export const AutoTextArea = (props: TextareaProps & { inputref: RefObject<unknown> }) => {
```

Comment:
yes was fighting a bit with this
the ref is in the parent because that is where the useReactiveTextInput is and that needs it also
could not find out a way to pass it backwards?

Reply:
Makes sense, thank you for explaining